errbot/backends/hipchat.py

```diff
@@ -442,7 +442,7 @@ def query_room(self, room):
         """
         if room.endswith('@conf.hipchat.com'):
             log.debug("Room specified by JID, looking up room name")
-            rooms = self.conn.hypchat.rooms(expand='items')
+            rooms = self.conn.hypchat.rooms(expand='items').contents()
```

Comment:
This fix is not symmetric with the other one, I guess one or the other is buggy ?
Here we to rooms['items'] but on the other we don't ?